### DISCUSSION TITLE
Postgres delete row

### DISCUSSION THREAD

**Discussion by damacchiavello** [Budibase/budibase#5601] [Help]
I have a simple table "Companies" in Postgres.
When I try to use the delete button, it returns to the Companies list as expected, but it does not delete the row.
What values I have to complete in the fields "Row ID" and "Row Rev" ?
I can't delete a row !

![image](https://user-images.githubusercontent.com/102870074/165406494-0c47e489-7aba-402b-a8cd-3bf0bab57a6d.png)

**Comment by melohagan** [comment]
Hi @damacchiavello 

You need to enter a binding into the 'Row Rev' field. It will look something like: `{{ Repeater.Companies._rev }}`

In future, this Row Rev field will be removed, and will be set automatically so the user doesn't need to worry about it.
